docs/examples/output_parsing/GuardrailsDemo.ipynb

```diff
@@ -30,7 +30,11 @@
     "logging.getLogger().addHandler(logging.StreamHandler(stream=sys.stdout))\n",
     "\n",
     "from llama_index import VectorStoreIndex, SimpleDirectoryReader\n",
-    "from IPython.display import Markdown, display"
+    "from IPython.display import Markdown, display\n",
+    "\n",
+    "import openai\n",
+    "\n",
+    "openai.api_key = \"<YOUR_OPENAI_API_KEY>\""
```

Comment:
we can probably leave these out, to be passed in via env vars

Reply:
Also, the llm starts downloading llama.cpp if you are using another llm like anyscale. You get an error like this, i believe its because of langchain hardcoded defaults that overrides llamaindex llm setting. 
 ```
 File "llamas.py", line 33, in <module>
    llm_predictor = StructuredLLMPredictor()
  File "/Users/mac/Documents/GitHub/datagpt/pluralsh/dev/lib/python3.8/site-packages/llama_index/llm_predictor/base.py", line 88, in __init__
    self._llm = resolve_llm(llm)
  File "/Users/mac/Documents/GitHub/datagpt/pluralsh/dev/lib/python3.8/site-packages/llama_index/llms/utils.py", line 40, in resolve_llm
    llm = LlamaCPP(
  File "/Users/mac/Documents/GitHub/datagpt/pluralsh/dev/lib/python3.8/site-packages/llama_index/llms/llama_cpp.py", line 90, in __init__
    raise ImportError(
ImportError: Could not import llama_cpp library.Please install llama_cpp with `pip install llama-cpp-python`.See the full installation guide for GPU support at `https://github.com/abetlen/llama-cpp-python`
```

Reply:
Yes. LlamaIndex either wants an OpenAI API key setup or it will fallback to open-source models. Have a look here: https://gpt-index.readthedocs.io/en/stable/getting_started/installation.html#openai-environment-setup